src/components/shared/CardList/Card/Card.jsx

```diff
@@ -17,10 +17,17 @@ export class Card extends React.Component {
         if (this.props.enabled) this.props.handleClick(this.props.card.cardId);
     }
 
+    getImageForCard() {
+        if (this.props.hidden) {
+            return require(`../../../../images/cardBack.jpg`);
+        }
+        return require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`);
+    }
+
     render() {
         return (
             <div data-cy='card-wrapper' className={cx(styles.cardWrapper, { enabled: this.props.enabled }, { disabled: !this.props.enabled }, {selected : this.props.playedCardId === this.props.card.cardId})} onClick={this.handleClick}>
-                <img data-cy='card' className={styles.card} alt={`card-${this.props.card.cardId}`} src={require(`../../../../images/cards/card (${this.props.card.cardId}).jpg`)} />
+                <img data-cy='card' className={styles.card} alt={`card-${(this.props.hidden) ? "hidden" : this.props.card.cardId}`} src={this.getImageForCard()} />
```

Comment:
I think if you really want to hide the cardId from people then it's better off not actually sending the cards in the first place, and just have the server tell the client how many face down cards to render, because they are still going to be stored in redux and the user could also print out their socket stuff and see it